src/style.css

```diff
@@ -1,3 +1,154 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  padding: 32px 19px 19px;
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  align-items: center;
+  flex-shrink: 0;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  /* margin-bottom: 40px; */
+  flex-shrink: 0;
+}
+
+.card__content {
+  margin-top: 28px ;
+}
+
+.card__name {
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  font-family: "Roboto", sans-serif;
+}
+
+.card_code {
+  color: #616070;
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  margin-top: 4px;
+  text-align: left;
+  font-family: "Roboto", sans-serif;
+}
+
+.card__raiting {
+  display: flex;
+  width: 166px;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-top: 16px;
+}
+
+.stars {
+  width: 96px;
+  height: 16px;
+  display: flex;
+  flex-shrink: 0;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 5px;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(1n) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__raiting-review {
+  display: flex;
+  text-align: right;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card__price {
+  width: 166px;
+  display: flex;
+  justify-content: space-between;
+  font-size: 12px;
+  line-height: 16px;
+  color: #616070;
```

Comment:
use variables for repetitive values like colors

Reply:
<img width="791" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/102411130/89185c27-bf77-4414-9f36-3a32aedc1701">